django-rgd/rgd/utility.py

```diff
@@ -294,17 +294,34 @@ def clean_file_cache(override_target=None):
             )
             break
         path = paths.pop(0)
-        if os.path.isdir(path):
-            shutil.rmtree(path)
-        else:
-            os.remove(path)
+        # Check if resource is locked and skip if so
+        lock_path = Path(f'{path}.lock')
+        lock = FileLock(lock_path)
+        try:
+            with lock.acquire(timeout=0.000000001):
```

Comment:
Filelock's docs state that you probably just want 0 here (https://py-filelock.readthedocs.io/en/latest/api.html#filelock.BaseFileLock.acquire)

Reply:
Ah good catch, thanks!